hydra-filters/src/test/java/com/addthis/hydra/data/filter/bundle/TestBundleFilterTimeRange.java

```diff
@@ -13,38 +13,69 @@
  */
 package com.addthis.hydra.data.filter.bundle;
 
+import com.addthis.bundle.core.Bundle;
 import com.addthis.bundle.util.AutoField;
+import com.addthis.codec.config.Configs;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 import org.joda.time.format.DateTimeFormatter;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.io.IOException;
 import java.text.DecimalFormat;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
 import static org.mockito.Mockito.mock;
 
 public class TestBundleFilterTimeRange {
     AutoField timeIn ;
     DateTime utc;
     DecimalFormat df;
+    Bundle bundle;
+
+    static final String before1 = "time:TIME, before:2016-09-23, timeFormat:YYYY-MM-dd";
+    static final String before2 = "time:TIME, before:2015-09-25, timeFormat:YYYY-MM-dd";
+    static final String after1  = "time:TIME,  after:2016-09-21, timeFormat:YYYY-MM-dd";
+    static final String after2  = "time:TIME,  after:2017-09-21, timeFormat:YYYY-MM-dd";
 
     @Before
-    public void before () {
+    public void before() throws IOException {
         this.timeIn = mock(AutoField.class);
         utc = new DateTime(DateTimeZone.UTC);
         df = new DecimalFormat("00");
+        bundle = Configs.decodeObject(Bundle.class, "TIME:1474502400000");   // 09/22/2016
+    }
+
+    @Test
+    public void testBeforeNoTimeZone() throws IOException {
+        BundleFilterTimeRange filter1 = Configs.decodeObject(BundleFilterTimeRange.class, before1);
+        BundleFilterTimeRange filter2 = Configs.decodeObject(BundleFilterTimeRange.class, before2);
+        assertTrue(filter1.filter(bundle));
+        assertFalse(filter2.filter(bundle));
     }
 
     @Test
-    public void testBundleFilterTimeRangeAB() {
+    public void testAfterNoTimeZone() throws IOException {
+        BundleFilterTimeRange filter1 = Configs.decodeObject(BundleFilterTimeRange.class, after1);
+        BundleFilterTimeRange filter2 = Configs.decodeObject(BundleFilterTimeRange.class, after2);
+        assertTrue(filter1.filter(bundle));
+        assertFalse(filter2.filter(bundle));
+    }
+
+    @Test(expected= Exception.class)
```

Comment:
I assume it is a subclass of Exception expected here? If so, you should use the exact type, not the catchall `Exception`.